versioned_docs/version-2.0/how_to_guides/evaluation/filter_experiments_ui.mdx

```diff
@@ -0,0 +1,93 @@
+import {
+  CodeTabs,
+  python,
+  typescript,
+  PythonBlock,
+  TypeScriptBlock,
+} from "@site/src/components/InstructionsWithCode";
+
+# Filter experiments in the UI
+
+LangSmith lets you filter your previous experiments by feedback scores and metadata to make it easy
+to find only the experiments you care about.
+
+## Background: add metadata to your experiments
+
+When you run an experiment in the SDK, you can attach metadata to make it easier to filter in UI. This
+is helpful if you know what axes you want to drill down into when running experiments.
+
+In our example, we are going to attach metadata to our experiment around the model used, the model provider,
+and a known ID of the prompt:
+
+<CodeTabs
+  groupId="client-language"
+  tabs={[
+    python`
```

Comment:
needs JS still

Reply:
actually, nvm, this isn't that important to show in JS and probably better uses of time. Can add later